src/js/components/EditsTable.jsx

```diff
@@ -52,8 +50,17 @@ export const renderTableCaption = (edit, rowObj, pagination) => {
   const editText = length === 1 ? 'edit' : 'edits'
   const captionHeader = `${length} ${name} ${editText} found.`
 
+  if(type === 'macro') {
+    return (
+      <div className="caption">
+        <h3>{captionHeader}</h3>
+        {description ? <p className="usa-font-lead">{description}</p>:null}
+      </div>
+    )
+  }
+
   return (
-    <caption>
+    <caption className="caption">
```

Comment:
Is the `className` needed here? (And [above](https://github.com/cfpb/hmda-platform-ui/pull/490/commits/aa6ec7be754e477479a18f552da12af63ef1d465#diff-d1bd5067e44fa2ba9bdd347727b4fd51R55))